scripts/coreos-disable-fips

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+# Copyright (c) 2019 Red Hat, Inc.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+set -euo pipefail
+
+RPM_OSTREE_BIN=$(which rpm-ostree)
+
+# check_and_disable looks to see if the kernel arg is already there. If it is
+# there, the argument will be delete.
+function check_and_disable() {
+    CHECK_FOR_ARG=$1
+    # Remove the kernel arg if it exists
+    set +e
+    ${RPM_OSTREE_BIN} kargs | grep "${CHECK_FOR_ARG}" > /dev/null
+    ARG_CHECK=$?
+    set -e
+    if [[ ${ARG_CHECK} == 0 ]]; then
+        ${RPM_OSTREE_BIN} kargs --delete=${CHECK_FOR_ARG}
+    fi
+}
+
+# is_ostree_system checks /etc/os-release for ostree variables
+# in an attempt to verify this is an ostree owned system
+function is_ostree_system() {
+    set +e
+    grep OSTREE_VERSION /etc/os-release > /dev/null
+    HAS_OSTREE_VERSION=$?
+    if [[ ${HAS_OSTREE_VERSION} != 0 ]]; then
+        echo "OSTREE_VERSION not found in /etc/os-release."
+        echo "This does not seem to be an ostree owned system."
+        echo "Exiting without any changes."
+        exit 1
+    fi
+    set -e
+}
+
+
+# If FIPS is already disabled, exit out quickly
+if [ `cat /proc/sys/crypto/fips_enabled` == 0 ]; then
+    exit 0
+fi
+
+# Ensure we are on an ostree based system
+is_ostree_system
+# Get the boot disk UUID
+UUID=`blkid -l -s UUID -t LABEL=boot -o value`
```

Comment:
Use `$(...)` instead of backticks

https://github.com/koalaman/shellcheck/wiki/SC2006

Reply:
Updated